### PR TITLE
pounce: update to 2.4.

### PR DESCRIPTION
**Pull request by paper42**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/pounce/template [hunk 1]
```diff
@@ -1,14 +1,15 @@
 # Template file for 'pounce'
 pkgname=pounce
-version=2.3
-revision=2
-build_style=gnu-configure
+version=2.4
+revision=1
+build_style=configure
```
Comment (ericonr) [inline srcpkgs/pounce/template:5]: Please explain the switch from `gnu-configure` in the commit message. AFAIK June's stuff carried "autoconf compatible" configure scripts.

Comment (paper42) [reply srcpkgs/pounce/template]: I didn't look into it enough, sorry, reverted.

**Comment by paper42** [comment]
I added an openssl runtime dependency, because pounce -g execs openssl to generate a certificate and libtls doesn't depend on openssl.

File: srcpkgs/pounce/template [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 pkgname=pounce
-version=2.3
-revision=2
+version=2.4
+revision=1
 build_style=gnu-configure
 make_build_target="all"
 hostmakedepends="pkg-config"
-makedepends="libtls-devel"
+makedepends="libtls-devel openssl-devel"
```
Comment (ericonr) [inline srcpkgs/pounce/template:8]: Why have `openssl-devel`?

Comment (paper42) [reply srcpkgs/pounce/template]: Falling back to /bin/openssl is fine on void, removed.

**Pull request merged**